### ISSUE TITLE
goToStep not working for multiple times for special values

### ISSUE THREAD

**Issue by schuesslerf** [usablica/intro.js#601]
Hello,

I figured out a possible bug (or I'm doing something wrong, what I don't expect in that case). 

What I'm currently trying to do with IntroJS is to show only one specified step and then end the tour (with a OK button, or by just clicking out of the step area). 

Therefore I found your function goToStep(), which in generally works, so what I do is define the tour, and then providing two functions:

```
 startIntro() {
        this.intro.setOption('showButtons',true);
        this.intro.start();
    }

    showStep(step){
        this.intro.setOption('showButtons', false);
        this.intro.goToStep(step).start();
    }
```

And in general it works, clicking the button that's calling startIntro() "plays" the complete tour and clicking the button that's calling showStep() shows only the one step specified in parameter (or at least it looks like theres only one step shown, actually you just can't move to another step).
**But:** once I try to go to Step 1 or 2 (or in step counter 0 or 1) it doesn't work properly, only showing step 2 (1 starting to count from 0) works **once**, when clicking the button again it doesn't work any more. Only showing step 1 (0 starting to count from 0) doesn't work at all.

I'm getting the errors 
Cannot read property 'nodeName' of null
and 
Cannot read property 'element' of undefined

So it doesn't seem to find the DOM elements any more.

I think (pure guessing, don't know your code good), that this has something to do with your "goToStep()" function, especially that line:

`this._currentStep = step - 2;`

This could propably explain that error.

Is there nevertheless maybe something wrong in my code, or I have a wrong way of thinking, or is this a bug of your plugin? If yes, is there any workaround you can think so far?
`

**Comment by aliel** [comment]
Hi

I have the same problem

**Comment by radumpopescu** [comment]
From what I saw in the latest commits (39e49d63f6f9b582f5b4169e2ed775005289a3ff) the method `goToStepNumber` has been added but that's not available in the released 2.4.0 version.

**Comment by afshinm** [comment]
Will be released for the next version.
